torch/csrc/jit/codegen/cuda/scheduler.cpp

```diff
@@ -385,7 +369,9 @@ ReductionParams reductionHeuristic(
 } // anonymous namespace
 
 // fusion is the input IR that will be modified by this function
-bool scheduleReduction(Fusion* fusion, const at::ArrayRef<c10::IValue> inputs) {
+c10::optional<ReductionParams> scheduleReduction(
+    Fusion* fusion,
+    const at::ArrayRef<c10::IValue> inputs) {
```

Comment:
was the intention `const at::ArrayRef<c10::IValue>&` ?

Reply:
It should have been.  Thanks for the catch!